sydneywait/AppalachianHarvest
# Add a producer to the database

# Description
Users can add a producer to the database

## Related Ticket(s)
https://github.com/sydneywait/AppalachianHarvest/issues/3


## Steps to Test
--Add and commit on your own branch before checking out to sw-add-producers
--Click on 'producers' in the nav bar
--You'll see a display of the producers in the database shown in card format
--Click on 'add a producer' and enter in the information.  All fields are required except for the image.  The zipcode, phone number, and state abbreviation have specific formats.
--click 'submit' and you will be redirected to the index view with all the producers.

# Checklist:
